server/api/router.go

```diff
@@ -17,10 +17,13 @@ package api
 import (
```

Comment:
Please change the permission of this file back.

Reply:
changed